source/client/ClickableImage.py

```diff
@@ -0,0 +1,39 @@
+import pygame
+import client.UIConstants as UIC
+
+
+class ClickableImage:
+    """This class draws an image which can be used as a button to initiate actions.
+
+    Edited code for class button found on-line at
+    https://www.youtube.com/watch?v=4_9twnEduFA
+    to create this.
+    """
+    def __init__(self, image, x, y, width, height, outline_index):
+        self.image = image
```

Comment:
Maybe make an option for this where you can do an init with just image, x, y, and outline_index and have init call get_width and get_height on the image to simplify the call?